ioloop.lisp

```diff
@@ -66,9 +66,8 @@
 ;;;; multiplexer to actually run it.
 
 (export '(io-channel-ioport io-channel-events io-channel-handle
-          *default-io-loop* *current-io-loop* *current-io-channel*
           io-loop io-loop-add io-loop-remove io-loop-update
-          callback-channel callback-channel-stream callback-channel-events))
+          *default-io-loop* *current-io-loop*))
 
 ;;; General interface
 (defgeneric io-channel-ioport (io-loop channel)
```

Comment:
It looks like you changed the expected return type of this method? Can you update the documentation string if so?

Reply:
Huh, what method?

Reply:
A couple of implementations of this method changed, and I was wondering if there was an update to how this method is used that required a documentation update.